modules/jupyter-widget/src/utils.js

```diff
@@ -0,0 +1,112 @@
+/* global document */
+/**
+ * Gets CSS from a given URL and appends it to the document head
+ * @param url URL of CSS page
+ */
+function loadCss(url) {
+  const link = document.createElement('link');
+  link.type = 'text/css';
+  link.rel = 'stylesheet';
+  link.href = url;
+  document.getElementsByTagName('head')[0].appendChild(link);
+}
+
+/**
+ * Creates a div of 500px by 500px for the Jupyter widget
+ * @param idName ID attribute of div
+ * @return div object for appending to the document
+ */
+function createWidgetDiv(idName, width = 500, height = 500) {
+  const div = document.createElement('div');
+  div.style.width = `${width}px`;
+  div.style.height = `${height}px`;
+  div.id = idName;
+  return div;
+}
+
+/**
+ * Creates a canvas for the Jupyter widget
+ * @param idName ID attribute of canvas
+ * @return canvas canvas for appending
+ */
+function createCanvas(idName) {
+  const canvas = document.createElement('canvas');
+  Object.assign(canvas.style, {
+    width: '100%',
+    height: '100%',
+    position: 'absolute',
+    'z-index': 2
+  });
+  canvas.id = idName;
+  return canvas;
+}
+
+/**
+ * Creates a div for the Jupyter widget's basemap
+ * @param idName ID attribute of div
+ * @return div object for appending to the document
+ */
+function createMapDiv(idName) {
+  const div = document.createElement('div');
+  Object.assign(div.style, {
+    'pointer-events': 'none',
+    height: '100%',
+    width: '100%',
+    position: 'absolute',
+    'z-index': 1
+  });
+  div.id = idName;
+  return div;
+}
+
+/**
+ * Hides a warning in the mapbox-gl.js library from surfacing in the notebook as text.
+ */
+function hideMapboxCSSWarning() {
```

Comment:
Can't we solve this by making sure we import the CSS?

Reply:
Tried this and couldn't get it to work on my machine but I'm open to that possibility.